### PR TITLE
feat(launch event): #62 ready for test

### PR DESCRIPTION
**Pull request by DiLiUA**
- moved artist name to first step;
  - added question about crowdcampaign;
  - added functionality for time zone;

### CODE REVIEW

File: src/event-launch/event-launch.component.html [hunk 1]
```diff
@@ -46,7 +46,39 @@
...
+                       (click)="changeCrowfundingType(campaignType)">
+                  {{campaignType.title}}
+                </label>
+              </div>
+            </div>
+
+            <div class="form-group" *ngIf="checkedCrowdCampaignType === crowdcampaignForSomeOtherArtist">
+              <label class="field-title">Band or Artist name:</label>
+              <input type="text"
+                     id="artist-name"
```
Comment (Ugmaxie) [inline src/event-launch/event-launch.component.html]: d'you need id here?

File: src/shared/show-service/show.service.ts [hunk 2]
```diff
@@ -4,10 +4,13 @@ import { Observable } from 'rxjs/Observable';
 import { catchError } from 'rxjs/operators';
 import { map } from 'rxjs/operators';
 
+let moment = require('moment-timezone');
```
Comment (Ugmaxie) [inline src/shared/show-service/show.service.ts]: no require

File: src/show-page/show-page.component.ts [hunk 3]
```diff
@@ -48,9 +50,9 @@ export class ShowPageComponent implements OnInit, OnDestroy {
   }
 
   getCurrentShow(rawQuery: {[key: string]: any}): void {
```
Comment (Ugmaxie) [inline src/show-page/show-page.component.ts]: avoid any

**Review by Ugmaxie — CHANGES_REQUESTED** [review]

File: src/event-launch/event-launch.component.html (hunk 1 above)
Comment (DiLiUA) [reply src/event-launch/event-launch.component.html]: yes. I added `for` attribute for `label` tag

**Review by Ugmaxie — APPROVED** [review]

**Pull request merged**